src/database.js

```diff
@@ -128,6 +128,32 @@ function setUpModels(db) {
             }
         }
     });
+    models.Pod.homeStats = function (callback) {
+        db.driver.execQuery(
+          "SELECT \
+            (SELECT COUNT(*) FROM pods) AS total_nodes,\
```

Comment:
>We should probably add indexes to the tables, starting to have lots of data and more selects to render the page, I'll make a separate issue re that.

Yeah I want to refactor the backend: we don't need to process the data at each rendering: they changed only once per day so we could do the maths when inserting the new data. In fact if we would like to be perfectly optimized, we could generate the HTML at this time and render only plain HTML time :P No need to go that far but we'll surely improve the database part in the future.